ravencoin/txscript/data/script_tests.json

```diff
@@ -0,0 +1,1882 @@
+[
```

Comment:
at *some* point we should probably update this file, but I would call it low-priority as this isn't related to the rosetta testing, just a test imported from the btcd repo.